code/go/0chain.net/chaincore/chain/handler.go

```diff
@@ -1699,7 +1701,7 @@ func StateDumpHandler(w http.ResponseWriter, r *http.Request) {
 
 	if contract == "" {
 		contract = "global"
-	} else {
+	} else { //nolint: staticcheck
 		//TODO: get the smart contract as an optional parameter and pick the right state hash
```

Comment:
This todo was added [3 years ago](https://github.com/0chain/0chain/blame/staging/code/go/0chain.net/chaincore/chain/handler.go#L1700), I think it is no longer of use to whoever put it there. If necessary we should replace with a comment before the `if` statement and remove the empty if block.